docs/src/main/docs/tutorials/testing-rpc-service.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: docs
+title: Testing an RPC service
+section: tutorials
+permalink: tutorials/testing-rpc-service
+---
+
+# Tutorial: Testing an RPC service
+
+This tutorial will show you how to write a unit test for an RPC service using an
+in-memory channel and client.
+
+This tutorial is aimed at developers who:
+
+* are new to Mu-Scala
+* have some understanding of Protobuf and `.proto` file syntax
+* have read the [Getting Started guide](../getting-started)
+* have followed the [gRPC server and client tutorial](grpc-server-client)
+
+## Service definition
+
+Let's use the following service definition. (For an explanation of how to create
+service definition, check out the [RPC service definition with Protobuf
+tutorial](service-definition/protobuf).)
```

Comment:
it might be worth mentioning that while this guide is using Protobuf, the notions covered are also useful for Avro users, wdyt?

Reply:
maybe in the section above